ideascube/serveradmin/management/commands/csv2pkg.py

```diff
@@ -21,18 +22,37 @@ def add_arguments(self, parser):
         parser.add_argument('--dry-run', action='store_true',
                             help='Do not really create the package.')
 
-    def abort(self, msg):
+    def abort(self, msg=""):
         self.stderr.write(msg)
         sys.exit(1)
 
     def handle(self, *args, **options):
+        self.report = Reporter(options['verbosity'])
         csv_path = os.path.abspath(options['csv_path'])
         package_path = os.path.abspath(options['package_path'])
 
+        if options['dry_run']:
+            print("--dry-run option given, no package will be created")
+
         if not os.path.exists(csv_path):
             self.abort('Path does not exist: {}'.format(csv_path))
         os.makedirs(os.path.dirname(package_path), exist_ok=True)
 
-        package = MediaCenterPackage.from_csv(csv_path)
-        if not options['dry_run']:
+        package = MediaCenterPackage.from_csv(csv_path, report=self.report)
+        print(self.report)
+
+        if not package.medias:
+            print("There is no (valid) media to create the package.\n"
+                  "No package will be created")
+            self.abort()
```

Comment:
I would be of bad faith, I would say to print in stdout instead of stderr.

But the truth is that I forget the `self.abort` and add it later. I will pass the message to `abort`